### ISSUE TITLE
Obsoletion request: GO:0006379	mRNA cleavage, GO:0031439 positive regulation of mRNA cleavage, GO:0031437	regulation of mRNA cleavage, GO:0031438	negative regulation of mRNA cleavage, GO:0098787	mRNA cleavage involved in mRNA processing

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#25185]
Please provide as much information as you can:

* **GO term ID and Label**

GO:0006379	mRNA cleavage
REPLACE BY -> GO:0031124  mRNA 3'-end processing

GO:0031439	positive regulation of mRNA cleavage
REPLACE BY -> GO:0031442    positive regulation of mRNA 3'-end processing

GO:0031437	regulation of mRNA cleavage
REPLACE BY -> GO:0031440    regulation of mRNA 3'-end processing

GO:0031438	negative regulation of mRNA cleavage
REPLACE BY -> GO:0031441    negative regulation of mRNA 3'-end processing

GO:0098787	mRNA cleavage involved in mRNA processing
REPLACE BY -> GO:0031124  mRNA 3'-end processing

* **Reason for deprecation** Put an x in the appropriate box: 

 - [x] The reason for obsoletion is that this term represents a molecular function.

* **"Replace by" term (ID and label)**
If all annotations can safely be moved to that term

See above

* **Are there annotations to this term?**
- How many EXP:

* **Are there mappings and cross references to this term? (InterPro, Keywords; check QuickGO cross-references section)**

* **Is this term in a subset? (check the AmiGO page for that term)**

* **Any other information**

----
Checklist for ontology editor

***Check term usage and metadata in Protégé***
- [ ] check term usage in the ontology 
- [ ] check internal mappings: RHEA, EC, MetaCyc
- [ ] check subset usage
- [ ] check taxon constraints 

***Check annotations***

Annotation review not required. annotation will mgrate with 'replaced by' tag.

***Notification***
- [ ] create [obsoletion announcement](https://github.com/geneontology/go-announcements/issues/new?assignees=&labels=obsoletion&template=obsoletion-notice.md&title=Obsoletion+notice%3A+%5BGO+ID%3A+term+label%5D)
- [ ] announce to GO friends (go-friends@mailman.stanford.edu)
- [ ] paste the text in the ontology ticket

**Comment by ValWood** [comment]
obsolete on or after 05-04-23

**Comment by RLovering** [comment]
maybe try using QuickGO to check for these. The URL below gives 75 annotations

https://www.ebi.ac.uk/QuickGO/annotations?goId=GO:0006379&goUsageRelationships=is_a,part_of,occurs_in,regulates&goUsage=descendants&evidenceCode=ECO:0000269,ECO:0000304,ECO:0000303&evidenceCodeUsage=descendants

and includes the term:
pre-mRNA cleavage required for polyadenylation

Also are you sure that GO:0006379 mRNA cleavage can be replaced by GO:0031124 mRNA 3'-end processing. Rebecca had annotated ERN1 to mRNA cleavage involved in mRNA processing based on its role in: mRNA splicing, via endonucleolytic cleavage and ligation.

Thanks

Ruth

**Comment by ValWood** [comment]
Right, mRNA cleavage  cant be migrated to 3' end-processing (I thought all of these were used in the context of 'cleavage and polyadenylation"
I will need to obsolete  without a direct replacement so people will need to reannotate after all.

For 
37 GO:0098789	pre-mRNA cleavage required for polyadenylation
I'm going to remove the "RNA hydrolysis" parent from this term, and deal with it when I do the "polyadenylation branch", so I think the numbers might be correct?

**Comment by ValWood** [comment]
I only found  8 annotations
https://github.com/geneontology/go-annotation/issues/4517
(I'm excluding 
GO:0098787 mRNA cleavage involved in mRNA processing
 because those can still be re-mapped automatically)

**Comment by pfey03** [comment]
We only had 3 very old ISS and I deleted those; didn't find good replacements.

**Comment by ValWood** [comment]
Hi Petra

For anything which is
GO:0005849   mRNA cleavage factor complex
The correct BP term would be
GO:0180010   co-transcriptional mRNA 3'-end processing, cleavage and polyadenylation pathway

At the moment I'm weeding out the unnecessary MF terms  but I'll provide a schema to map
complexes to processes, but the terms  we are removing aren't mapping to the specific processes.

**Comment by pfey03** [comment]
Since we do not have any experimental annotations, no papers for these genes, I can only do ISS if available, and there is not much available as the orthologs also are still annotated to terms that will be obsoleted.
